### PR TITLE
Update validations for Search Bar fields

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#11810

*Description of changes:*

- Update MHR Number search value hint message
- Update validations per the validation matrix
- Update Special Character usage message for Organization search

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/utils/validation-helper.ts [hunk 1]
```diff
@@ -57,31 +57,31 @@ export function validateSearchAction (
     case APISearchTypes.INDIVIDUAL_DEBTOR:
       if (!first) {
         validation.searchValue.messageFirst = 'Enter a first name'
-      } else if (first?.length > 50) {
-        validation.searchValue.messageFirst = 'Maximum 50 characters'
+      } else if (first?.length > 15) {
+        validation.searchValue.messageFirst = 'Maximum 15 characters'
```
Comment (cameron-eyds) [inline ppr-ui/src/utils/validation-helper.ts]: Just to clarify, the task was to update PPR validations as well? I believe Debtor name is a PPR field.
Just double checkin :)

File: ppr-ui/src/utils/validation-helper.ts [hunk 2]
```diff
@@ -244,6 +244,10 @@ export function validateSearchRealTime (
       if (searchValue?.length > 70) {
         validation.searchValue.message = 'Maximum 70 characters'
       }
+      if (specialCharsLax.test(searchValue)) {
+        validation.searchValue.message =
+          "Names don't normally contain special characters"
```
Comment (cameron-eyds) [inline ppr-ui/src/utils/validation-helper.ts]: Single quotes here

**Review by cameron-eyds — COMMENTED** [review]
Looks good, just the one question!

File: ppr-ui/src/utils/validation-helper.ts (hunk 1 above)
Comment (doug-lovett) [reply ppr-ui/src/utils/validation-helper.ts]: Yes individual debtors are PPR parties, and the first, middle, last names can each be 50 characters.

Comment (dimak1) [reply ppr-ui/src/utils/validation-helper.ts]: Good catch, thanks! Let me fix it.

Comment (dimak1) [reply ppr-ui/src/utils/validation-helper.ts]: Reverted back to 50.

File: ppr-ui/src/utils/validation-helper.ts (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/utils/validation-helper.ts]: Other strings with single quotes in words have double quotes, so I made the same.

**Comment by dimak1** [comment]
@cameron-freshworks addressed your comments

File: ppr-ui/src/utils/validation-helper.ts (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/utils/validation-helper.ts]: Fair enough, could you fix the other usages of double quotes to single quotes as well 👍

Comment (cameron-eyds) [reply ppr-ui/src/utils/validation-helper.ts]: Just to clarify, i didn't mean outside this file. 
Just ensure there is consistency in this file and we can tackle other instances as we go :)

Comment (cameron-eyds) [reply ppr-ui/src/utils/validation-helper.ts]: Excellent, thanks for the update!

**Review by cameron-eyds — APPROVED** [review]
Looks good! 
There was another PR merged, so lets wait for that build to resolve before merging 👍

**Review by doug-lovett — APPROVED** [review]

**Pull request merged**